package.json

```
@@ -17,11 +17,14 @@
     "spell": "cspell \"**/*.{ts,tsx,js,jsx,md,json}\"",
     "spell:check": "cspell \"**/*.{ts,tsx,js,jsx,md,json}\" --no-progress",
     "spell:fix": "cspell \"**/*.{ts,tsx,js,jsx,md,json}\" --show-suggestions",
+    "test": "jest",
```

Biome (1.9.4)

[error] The key test was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)